pkg/reconciler/dependencybuild/scripts/gradle-build.sh

```diff
@@ -27,6 +27,10 @@ RELEASE_REPOSITORY_URL=file:$(workspaces.source.path)/artifacts
 RELEASE_SIGNING_ENABLED=false
 mavenCentralUsername=
 mavenCentralPassword=
+
+# Default values for common enforced properties
+sonatypeUsername=jbs
+sonatypePassword=jbs
```

Comment:
As I've seen several projects including https://github.com/fge/btf/blob/1.2/build.gradle#L130 have checks for these variables it seems simpler to dummy up some values.